src/types/types.ts

```diff
@@ -46,12 +46,16 @@ export interface EventItemType {
 }
 
 export interface GridGalleryItemType {
-  image: MediaItemType
-  monthYear: string
-  to: string
-  headlineText: string
-  snippet: string
+  image?: MediaItemType
+  monthYear?: string
+  to?: string
+  headlineText?: string
+  snippet?: string
   featured?: boolean
+
+  // Options for FlexibleGridGalleryCards:
+  sectionTitle?: string
```

Comment:
do we need SectionTilte and sectionSummary here as we already have it in flexible types file